src/sonic-yang-models/yang-models/sonic-port.yang

```diff
@@ -51,6 +51,12 @@ module sonic-port{
 						length 1..128;
 					}
 				}
+                
+				leaf mode {
+					description "SwitchPort Modes possible values are routed|access|trunk. Default value  for mode is routed"; 
+					type stypes:switchport_mode; 
+					default "routed";
```

Comment:
The default value will introduce disruptive change. The current behavior is conditional "default value".